devlib/target.py

```diff
@@ -363,8 +363,9 @@ def get_directory(self, source_dir, dest, as_root=False):
 
     def execute(self, command, timeout=None, check_exit_code=True,
                 as_root=False, will_succeed=False):
```

Comment:
shouldn't Target.execute also provide strip_colors and forward it to the connection's `execute` ?

Reply:
I don't really mind although I tend to prefer not adding code that has absolutely no users. Not a huge deal, though.

Reply:
From what I can see, Target.execute is just a stub and all *Connection.execute take the same parameter list. Avoiding slight inconsistencies like that avoid these issues in the first place

Reply:
@setrofim , @marcbonnici : what do you guys prefer ? As long as this breakage gets resolved quickly I'm happy

Reply:
My vote would be to add in the extra parameter for consistency.

Reply:
Alright let's do that then :smile: